### PR TITLE
Updated content for buying other equipment screen and Income Tax references

### PR DESCRIPTION
**Pull request by stevenaproctor**
The cannotClaimBuyingEquipment needs a new paragraph that is 'guidance3 link.label2. guidance4 link.label3.' The links should continue the user on their journey and change their answer on the 'What are you claiming for?' screen to whatever link they select.

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -224,26 +224,32 @@ cannotClaimMileageFuelCosts.link.label = Read claim tax relief for your job expe
 
 cannotClaimBuyingEquipment.you.title = You cannot claim tax relief for these expenses
 cannotClaimBuyingEquipment.you.heading = You cannot claim tax relief for these expenses
-cannotClaimBuyingEquipment.you.guidance = You may be able to get an
+cannotClaimBuyingEquipment.you.guidance = For some larger items, like a computer, you may be able to claim a type of capital allowance called
+cannotClaimBuyingEquipment.you.guidance2 = You cannot claim capital allowances for buying a car, motorcycle or bicycle.
```
Comment (ShaolinSarg) [inline conf/messages.en]: i cant find a reference to this message key in the rest of the code, is it going to be added in a different pull request?

File: conf/messages.en [hunk 2]
```diff
@@ -224,26 +224,32 @@ cannotClaimMileageFuelCosts.link.label = Read claim tax relief for your job expe
...
 cannotClaimBuyingEquipment.someoneElse.title = They cannot claim tax relief for these expenses
 cannotClaimBuyingEquipment.someoneElse.heading = They cannot claim tax relief for these expenses
-cannotClaimBuyingEquipment.someoneElse.guidance = They may be able to get an
+cannotClaimBuyingEquipment.someoneElse.guidance = For some larger items, like a computer, they may be able to claim a type of capital allowance called 
+cannotClaimBuyingEquipment.someoneElse.guidance2 = They cannot claim capital allowances for buying a car, motorcycle or bicycle.
+cannotClaimBuyingEquipment.guidance3 = If the item is small and will last less than 2 years, claim tax relief for
```
Comment (ShaolinSarg) [inline conf/messages.en]: as above

**Comment by TheNickWilson** [comment]
Thanks @stevenaproctor - I've incorporated all these changes into the PR which has the backend code etc., so I'm closing this one
